src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="input-container">
+          <label>
+            Surname:
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off">
```

Comment:
If you have more than 2 attributes , you should add indentation for attributes, and put closing bracket on a new line& Fix it everywhere, so it looks like that:
```suggestion
            <input
              type="text"
              name="surname"
              autocomplete="off"
            >
```